src/nft/components/card/index.tsx

```diff
@@ -82,6 +84,7 @@ export const NftCard = ({
       testId={testId}
       onClick={() => {
         if (bagExpanded) setBagExpanded({ bagExpanded: false })
+        if (!doNotLinkToDetails) onNavigation?.()
```

Comment:
i think this prop is confusingly named. in the card it also has `<StyledLink to={doNotLinkToDetails ? '' : detailsHref}>` which would be a link to nowhere, which doesn't seem like the right semantic naming. it also seems like the click behavior should be fully encapsulated in onClick instead of having two different paths. @JackShort @cbachmeier looks like this was added here: #6048 - can you think of a way to improve this component API?

Reply:
i'm proposing we rename `doNotLinkToDetails` to `linkToDetails` (true by default), but we can wait for the NFT team if you want.

i think the onClick actually should be two separate paths - there is an onClick for the button and for the card itself. this probably wasn't clear from this PR, but at other callsites the card looks like this:

![image](https://user-images.githubusercontent.com/66155195/227643808-65b8f0c0-d814-4a7e-a0b7-cef007f3247e.png)

to make this clearer, i renamed `onClick` to `onButtonClick`.